app/scripts/components/boilerplate/dashboard-btn.jsx

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+
+require('stylesheets/boilerplate/buttons');
+
+const DashboardBtn = React.createClass({
+  render() {
+    return (
+      <div>
+      <a className="button-link" href="/waterpoints/" >water points</a>
```

Comment:
- Make sure HTML children are indented
- lets rename this to just "button", so `boilerplate/button.jsx`, `const Button = React....`, and `<div className="button">` for the main containing div.
- Internal links should use `<Link>` from "react-router", so `import { Link } from 'react-router';` at the top and `<Link to="/">` instead of `<a>`.
- We can make this button reuseable by putting `{this.props.children}` in the link instead of hard-coding "water points". We'll also need to take the URL as props. @gkalvatcheva if you want to have a quick call about how to do this I'm happy to.